timeboost/bid_validator.go

```diff
@@ -277,6 +277,23 @@ func (bv *BidValidator) fetchReservePrice() *big.Int {
 	return bv.reservePrice
 }
 
+// Check time-related constraints for bid.
+// It's useful to split out to be able to re-check just these contraints after
+// time has elapsed.
+func validateBidTemporal(roundTimingInfo *RoundTimingInfo, bidRound uint64) error {
```

Comment:
validateBidTimeConstraints feels more apt here

Reply:
Thanks that is a better name.